src/main/java/org/acme/nilshartmann/todo/domain/todo/TodoService.java

```diff
@@ -0,0 +1,32 @@
+package org.acme.nilshartmann.todo.domain.todo;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.List;
+
+@ApplicationScoped
+public class TodoService {
+
+    public List<Todo> getAllTodos() {
+        Todo todoObject1 = new Todo();
```

Comment:
Das Erzeugen der Test-Objekte bitte auslagern. Versuche den "Given"-Teil deiner Tests klein zu halten. Du wirst die To-Dos eventuell noch an anderer Stelle brauchen. Hier reicht es aber vielleicht aus, wenn du dir im Test eine Hilfsfunktion schreibst.

Reply:
Wobei an der Stelle kannst du es auch direkt mit @OnStartup oder im Konstruktor machen. So würde bei jedem Aufruf die Objekte enu aufgerufen werden.